src/Shared/Modifiers.cs

```diff
@@ -440,7 +440,8 @@ internal static string GetItemSpecModifier(string currentDirectory, string itemS
                         }
                         else
                         {
-                            modifiedItemSpec = Path.GetFileNameWithoutExtension(itemSpec);
+                            // Fix path to avoid problem with Path.GetDirectoryName when backslashes in itemSpec on Unix
```

Comment:
Haven't you already called `GetDirectoryName`? Should the comment say `GetFileNameWithoutExtension` or should the fixup be applied up a level?

Reply:
I'm not sure because GetDirectoryName handles a problem when called it on the root folder. As see in corefx implementation on Unix it happened only on "/". For this context "empty string", it is correct behavior for the filename. I can't find any way when it may be a problem. I try to make the minimum perf impact on my fix.

Reply:
I think @rainersigwald is saying that there might be a typo in the comment: `Path.GetFileNameWithoutExtension` instead of `Path.GetDirectoryName`. Is this correct?

Reply:
@cdmihai Sure, sorry I didn't catch up it.